xdsl/ir.py

```diff
@@ -1612,3 +1613,129 @@ def is_structurally_equivalent(
         ):
             return False
         return True
+
+
+class _AffineExprKind(Enum):
```

Comment:
I would prefer if we would move this outside of this file to be honest!
`ir.py` is already quite big, and it wouldn't hurt to move this a bit away!

Reply:
I agree. There is the affine dialect, right? Maybe move it there?